packages/components/Molecules/Tabs/Tabs.vue

```diff
@@ -66,6 +66,7 @@ $rpl-tabs-divider-border: 1px solid rpl-color('light_neutral') !default;
 
     &-icon {
       margin-left: $rpl-space;
+      fill: currentColor !important;
```

Comment:
Heya, do we need to use !important here? Use cases where the icon needs to be diffierent to the text colour will need complicated selectors to get around this

Reply:
yep that was a copypasta error - good pickup!